snowflake/main.go

```diff
@@ -14,8 +14,19 @@
 
 package main
 
-import "antrea.io/theia/snowflake/cmd"
+import (
+	"embed"
+
+	"antrea.io/theia/snowflake/cmd"
+	"antrea.io/theia/snowflake/pkg/infra"
+)
+
+// Embed the udfs directory here because go:embed doesn't support embeding in subpackages
+
+//go:embed udf/*
```

Comment:
nit: maybe we could rename the directory to `udfs/`?

Reply:
Addressed